src/Domain/DomainHelpers.hpp

```diff
@@ -112,6 +112,9 @@ enum class ShellWedges {
 /// This is done if `aspect_ratio` is set to something other than the default
 /// value of one. When the argument `use_logarithmic_map` is set to `true`,
 /// the radial gridpoints of the wedge map are set to be spaced logarithmically.
+/// The `number_of_layers` is used when the user wants to have multiple layers
+/// of Blocks in the radial direction. This is helpful for benchmarking
```

Comment:
Please remove the last sentence. I don't know what you mean by "useful for benchmarking"